up-core-api/uprotocol/uprotocol_options.proto

```diff
@@ -100,4 +114,26 @@ message UServiceTopic {
 
   // Name of the protobuf.Message
   string message = 3;
-}
\ No newline at end of file
+}
+
+// The following is used to identify files - to be transferred, or after transfer, to be accessed by the receiving uEs
+// File Proto
+message File {
```

Comment:
There is already a file.proto and it needs to be added to UAttributes as it is a header attribute.